CrewChiefV4/SpeechRecogniser.cs

```diff
@@ -806,10 +831,10 @@ void sre_SpeechRecognized(object sender, SpeechRecognizedEventArgs e)
             }
             else if (voiceOptionEnum == MainWindow.VoiceOptionEnum.ALWAYS_ON)
             {
-                sre.RecognizeAsyncStop();
```

Comment:
I think nAudio recording device should be conditional same as audio output.  Probably same preference will do

Reply:
Why was NAudio made conditional in the first place? Is there some disadvantage to using it?